src/Skoruba.IdentityServer4.Admin/Controllers/IdentityController.cs

```diff
@@ -135,7 +147,7 @@ public async Task<IActionResult> RoleUsers(string id, int? page, string search)
 
         [HttpPost]
         [ValidateAntiForgeryToken]
-        public async Task<IActionResult> UserProfile(TUserDto user)
+        public async Task<IActionResult> UserProfile(TUserDto user, String submit)
```

Comment:
Since this endpoint is submitted from the view via a "form post", the "Invite User" and the "Save User" button need to be distinguished.